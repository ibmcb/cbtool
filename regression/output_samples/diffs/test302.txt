 status: Sending a termination request for vm_39 (cloud-assigned uuid C93D148F-2EF6-547A-BFE9-3C485911DE87)....
 status: Sending a termination request for vm_113 (cloud-assigned uuid A0EF2ADF-AC0D-57E7-8876-522B6A4F3307)....
 status: Sending a termination request for vm_86 (cloud-assigned uuid 2E954108-8E72-5EAE-994A-1BF74604FC8A)....
 status: Sending a termination request for vm_116 (cloud-assigned uuid C48B5A78-ACA7-5839-831C-4075B1203D6B)....
 status: Sending a termination request for vm_115 (cloud-assigned uuid 17E5324B-17E2-5AC3-8E9B-C9CF1BD6BEE2)....
 status: Sending a termination request for vm_40 (cloud-assigned uuid 24E7A445-B566-58E4-A1A6-164FFEE6A404)....
 status: Sending a termination request for vm_85 (cloud-assigned uuid 31A1E656-E568-5D2A-9FCD-1BD6FB60AA4E)....
 status: Sending a termination request for vm_87 (cloud-assigned uuid 106CE9AD-B318-50A4-A33F-8CDD73AAF7DA)....
 status: Sending a termination request for vm_90 (cloud-assigned uuid 87ED2455-DB8D-56E3-B02F-C87D2F2133E4)....
 status: Sending a termination request for vm_84 (cloud-assigned uuid 3AA6C39D-F5A7-53EC-8F9E-E0E46E30FCCF)....
 status: Sending a termination request for vm_88 (cloud-assigned uuid 41DBB0CF-D48B-5B7B-8C25-3D465F6604ED)....
 status: Sending a termination request for vm_114 (cloud-assigned uuid BA50BD1C-A3B4-5A3A-B50E-6BA4DDFCF282)....
 status: Sending a termination request for vm_75 (cloud-assigned uuid 4543E31F-8FA2-5017-A73A-5294F0483FE9)....
 status: Sending a termination request for vm_89 (cloud-assigned uuid 477BE3BE-F0FE-51AC-A383-4B0E7B0DFD01)....
 status: Sending a termination request for vm_108 (cloud-assigned uuid 7E23BB27-08B9-5C43-8E63-A461705F3600)....
 status: Sending a termination request for vm_109 (cloud-assigned uuid 1B5F6A81-4611-5AAE-8234-FAE3206B154D)....
 status: Sending a termination request for vm_54 (cloud-assigned uuid 2AED1E22-7770-573A-A119-4987E9F83FA2)....
 status: Sending a termination request for vm_76 (cloud-assigned uuid CA459A32-7546-5C3D-9FEE-69F5A3EC942C)....
 status: Sending a termination request for vm_53 (cloud-assigned uuid 031C7DEA-A0F9-5F38-9FDA-F53F47511BE0)....
 status: Sending a termination request for vm_52 (cloud-assigned uuid 317650D7-F05B-589E-8613-30CB72B23970)....
 status: Sending a termination request for vm_79 (cloud-assigned uuid A7C62E00-CBD7-579B-90A5-B61094DA30A6)....
 status: Sending a termination request for vm_112 (cloud-assigned uuid 580BA9DE-DB6A-5A23-A7B4-FC8AB6383596)....
 status: Sending a termination request for vm_111 (cloud-assigned uuid F22C4E6C-A538-5C07-8335-5DE8CC94792B)....
 status: Sending a termination request for vm_110 (cloud-assigned uuid 1BC4A62E-6500-5F89-97B8-F9342B1C40CF)....
 status: Sending a termination request for vm_81 (cloud-assigned uuid 0B041450-578A-5770-BAB0-59F02276F52B)....
 status: Sending a termination request for vm_83 (cloud-assigned uuid 6CDECAA2-80EC-5A23-8368-77BAD0B36E72)....
 status: Sending a termination request for vm_47 (cloud-assigned uuid C1B90FA0-882F-5067-B45B-C9DD71BCC380)....
 status: Sending a termination request for vm_72 (cloud-assigned uuid 5839E1AB-A222-55E1-9FAF-346CDEC4E164)....
 status: Sending a termination request for vm_71 (cloud-assigned uuid 6A636B25-C6E4-54EB-B720-2857232E1C31)....
 status: Sending a termination request for vm_49 (cloud-assigned uuid DC08B1FF-463F-5D88-91EF-DD832257A227)....
 status: Sending a termination request for vm_80 (cloud-assigned uuid 2E368E86-0A11-5A49-9272-E8239C64B8BD)....
 status: Sending a termination request for vm_97 (cloud-assigned uuid 6B02CF1A-4F86-5397-887B-BFCE2CABBB6C)....
 status: Sending a termination request for vm_98 (cloud-assigned uuid FC95CFAA-14B6-516E-B838-DEC43D1A59B0)....
 status: Sending a termination request for vm_82 (cloud-assigned uuid F0CFC3AF-45DC-5BF5-BEAE-56A696883DF8)....
 status: Sending a termination request for vm_48 (cloud-assigned uuid BD48641C-A47B-5A71-800A-C14B0BFCC7A9)....
 status: Sending a termination request for vm_50 (cloud-assigned uuid 295E03EB-4F68-5D09-BEA1-632CD084A39E)....
 status: Sending a termination request for vm_51 (cloud-assigned uuid 779E9F75-3FBE-50E3-B329-A30E4137C511)....
 status: Sending a termination request for vm_92 (cloud-assigned uuid 37D483CB-A28D-53C7-B0C7-D15588C6CE32)....
 status: Sending a termination request for vm_93 (cloud-assigned uuid 2C389214-7878-5983-BEF2-D2371FB0CABD)....
 status: Sending a termination request for vm_94 (cloud-assigned uuid E28196F4-5435-57FA-ADCA-5B14D3FB5BCC)....
 status: Sending a termination request for vm_73 (cloud-assigned uuid BF1DDCF3-E3C2-56F0-98FC-AED67C95C3F9)....
 status: Sending a termination request for vm_103 (cloud-assigned uuid FF02AC3D-3C0E-5A87-8260-D33C029D863A)....
 status: Sending a termination request for vm_58 (cloud-assigned uuid 25BCFDF5-644A-51AB-A19C-A3E6D10F367C)....
 status: Sending a termination request for vm_95 (cloud-assigned uuid 54CB3BBC-49C5-5C46-A872-09D753E3EE8D)....
 status: Sending a termination request for vm_60 (cloud-assigned uuid 30D484DE-2788-5767-8F9D-6DC475BED9A1)....
 status: Sending a termination request for vm_74 (cloud-assigned uuid 7A79F0EA-056D-51AD-B6C4-6AEE4CDD4E08)....
 status: Sending a termination request for vm_96 (cloud-assigned uuid CD4D59C0-A009-537C-A3C4-4BE224FF7BD6)....
 status: Sending a termination request for vm_59 (cloud-assigned uuid 265B1DE7-A591-5164-BA36-3038A1DA512D)....
 status: Sending a termination request for vm_106 (cloud-assigned uuid C24A9E74-3825-5BBB-BD91-42E193CFB403)....
 status: Sending a termination request for vm_105 (cloud-assigned uuid ED19574A-5A48-58CD-AF99-FC00E37D2A9B)....
 status: Sending a termination request for vm_107 (cloud-assigned uuid 47E2F70A-383A-5E69-BF73-36B226E17A5B)....
 status: Sending a termination request for vm_104 (cloud-assigned uuid 1A44E97E-775C-51CD-AB87-35E631D66B13)....
|Used Memory                                          |23.15M                        
|Uptime (in seconds)                                  |2516861                       
|Total Connections Received                           |185705                        
|Total Commands Processed                             |11759411                      
|Storage Size                                         |18579456                      
|Data Size                                            |6796444                       
|Index Size                                           |2133936                       
|Average Object Size                                  |616.345696926                 
|AD93E38D-EAEC-5C15-BA20-7A27F63D13FF (simzone_b) VM RESERVATIONS|0                             
|E4B581D8-2DD3-5956-8196-B1B60C5DC5B0 (simzone_d) VM RESERVATIONS|0                             
|AD29082C-1389-58A3-BE64-3784D075F191 (simzone_c) VM RESERVATIONS|0                             
|75A875C6-F2C5-5C3D-BD34-051FBBE24C3A (simzone_a) VM RESERVATIONS|0                             
