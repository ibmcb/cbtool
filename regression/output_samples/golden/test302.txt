 status: Sending a termination request for vm_77 (cloud-assigned uuid 36EC2FFF-FF59-57CF-A059-5CDBBF447B40)....
 status: Sending a termination request for vm_39 (cloud-assigned uuid 03EAA2EE-132D-5B37-82FB-463E884A291A)....
 status: Sending a termination request for vm_40 (cloud-assigned uuid 7B96A71C-0CFB-53FF-961C-AFB8A2E879BA)....
 status: Sending a termination request for vm_78 (cloud-assigned uuid 2938EB94-BF54-5380-8158-0DBDCF56FD32)....
 status: Sending a termination request for vm_71 (cloud-assigned uuid 167E238B-0212-56BE-9563-A2C6561675DC)....
 status: Sending a termination request for vm_98 (cloud-assigned uuid CBE3B3E9-AACC-50B7-B2E0-9EB486049C8A)....
 status: Sending a termination request for vm_114 (cloud-assigned uuid D21CB7D8-DD04-5EBA-A545-0FD7C32CBBF5)....
 status: Sending a termination request for vm_76 (cloud-assigned uuid 4D071B28-C671-5C68-B079-667116DE224F)....
 status: Sending a termination request for vm_84 (cloud-assigned uuid 135C9007-2FEB-594C-8380-BE8AC8C966E9)....
 status: Sending a termination request for vm_90 (cloud-assigned uuid 568268AF-3FD2-5E82-A46A-13A8C0569CCA)....
 status: Sending a termination request for vm_115 (cloud-assigned uuid 7D9C0411-CD32-5371-A601-8159FCD16EB2)....
 status: Sending a termination request for vm_97 (cloud-assigned uuid 95AD07AC-82B2-5243-AB96-672C0079DEDA)....
 status: Sending a termination request for vm_85 (cloud-assigned uuid B0594AAC-7FE1-518C-ABAB-64E636C99F22)....
 status: Sending a termination request for vm_73 (cloud-assigned uuid 85DAB611-BF92-5872-8283-0080CDE9A94C)....
 status: Sending a termination request for vm_116 (cloud-assigned uuid 58767753-E2BE-5F08-9EA8-917D5B2286C0)....
 status: Sending a termination request for vm_72 (cloud-assigned uuid C321C6B2-07BD-5E99-AD83-C3DDDFA5799D)....
 status: Sending a termination request for vm_86 (cloud-assigned uuid E174BC26-B182-51B0-9BBF-F00325E9FF7A)....
 status: Sending a termination request for vm_74 (cloud-assigned uuid CBBEF4BF-A531-5B15-A6EA-575CD28B9621)....
 status: Sending a termination request for vm_75 (cloud-assigned uuid B6AEC616-B3B0-5B1B-9294-D8B4BA572169)....
 status: Sending a termination request for vm_113 (cloud-assigned uuid 8F622F09-CE4B-55C1-8CD7-C189A6656561)....
 status: Sending a termination request for vm_54 (cloud-assigned uuid C86CF2E5-65BF-59F3-A10F-4AE8314ECB96)....
 status: Sending a termination request for vm_52 (cloud-assigned uuid C690FF33-DC4C-5F01-A615-6DC83E34CAB2)....
 status: Sending a termination request for vm_104 (cloud-assigned uuid DF07A19B-FBDF-5BD2-8F30-D063505E008A)....
 status: Sending a termination request for vm_89 (cloud-assigned uuid 03C3833F-5723-5F15-B98C-3DC1FD2AEECC)....
 status: Sending a termination request for vm_88 (cloud-assigned uuid 6F73BEDE-926A-5761-8C8C-5BEB83161969)....
 status: Sending a termination request for vm_106 (cloud-assigned uuid B4B2F2F0-9873-55A9-A9F8-60B6D2AD7FF0)....
 status: Sending a termination request for vm_53 (cloud-assigned uuid 4565AC21-B269-5735-8597-78DD1208D1E3)....
 status: Sending a termination request for vm_92 (cloud-assigned uuid 3F81F8C7-908E-5952-8E6A-BA520422C7F4)....
 status: Sending a termination request for vm_105 (cloud-assigned uuid CD5114D3-0732-539B-9073-AE96FA3308A9)....
 status: Sending a termination request for vm_103 (cloud-assigned uuid B704CBE7-D827-5DA4-8BFB-AF9EDCF070DA)....
 status: Sending a termination request for vm_87 (cloud-assigned uuid 8A960E58-CD1B-57D0-89F4-70407CE1BD4A)....
 status: Sending a termination request for vm_107 (cloud-assigned uuid 4DEA5310-164C-5F15-A72D-FC37A9A7450D)....
 status: Sending a termination request for vm_93 (cloud-assigned uuid 69D24E42-5DCC-5F00-8819-3CAC0378A4E4)....
 status: Sending a termination request for vm_95 (cloud-assigned uuid 66CB85FA-4019-5286-812C-1B99279381A8)....
 status: Sending a termination request for vm_79 (cloud-assigned uuid 1580B64D-7614-5B2D-B0B3-3274E223F81D)....
 status: Sending a termination request for vm_96 (cloud-assigned uuid 1596ED4E-8601-5F5E-8CEF-2A87F8D46BC3)....
 status: Sending a termination request for vm_94 (cloud-assigned uuid 71463040-B34C-5C2C-9A6A-16C0E5B6FA9C)....
 status: Sending a termination request for vm_81 (cloud-assigned uuid F8CB78D9-7A5E-5833-8149-B2F8F88BE673)....
 status: Sending a termination request for vm_80 (cloud-assigned uuid 07EB6AAE-78CE-5CD6-BB99-67C3DBB944D2)....
 status: Sending a termination request for vm_82 (cloud-assigned uuid 8BE222D6-5F42-50CE-8D81-0EA93C1B37E3)....
 status: Sending a termination request for vm_58 (cloud-assigned uuid BF0ED057-1B17-5815-8A94-99F2E1227A7C)....
 status: Sending a termination request for vm_108 (cloud-assigned uuid FFFFB99D-C891-5D80-AEC1-8BAF08F6B1B8)....
 status: Sending a termination request for vm_60 (cloud-assigned uuid 6A633D50-F9DF-5DB8-BB74-CC4664D99B0A)....
 status: Sending a termination request for vm_83 (cloud-assigned uuid 52909C0F-D24C-5314-B617-71A03279CDCF)....
 status: Sending a termination request for vm_109 (cloud-assigned uuid ACDDB301-DA9A-5C3B-BE1B-28D3626C54A1)....
 status: Sending a termination request for vm_59 (cloud-assigned uuid C7445C1E-B3C6-51C2-A03F-B3BBAB402B6E)....
 status: Sending a termination request for vm_110 (cloud-assigned uuid 81283DC9-120B-58A8-9A86-8F7E3B00DC66)....
 status: Sending a termination request for vm_112 (cloud-assigned uuid 441ECE18-7822-5C3B-898C-8DBF8EA38627)....
 status: Sending a termination request for vm_111 (cloud-assigned uuid AACB200B-3F2F-5118-9EA4-DC7C48D81D1A)....
AI object undefined (named "all") was sucessfully detached from this experiment.
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
No objects available.
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |23.26M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2512148                       
|Total Connections Received                           |185429                        
|Total Commands Processed                             |11239357                      
|Number of Keys                                       |601                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18550784                      
|Data Size                                            |6054460                       
|Index Size                                           |1954064                       
|Average Object Size                                  |622.118783395                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |113                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |113                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |28                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |28                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |1                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |374                           

 ---------------- VMC-WIDE COUNTERS ----------------
|C4E90296-1043-55C8-8270-B5D7C3C38270 (simzone_b) VM RESERVATIONS|0                             
|2AEE56D4-E6A4-5FB2-8D29-4268D4C22AD0 (simzone_a) VM RESERVATIONS|0                             
|5F641DA9-DE4B-538C-B0CD-42A4F7D43E9E (simzone_d) VM RESERVATIONS|0                             
|A796D59F-B618-5C91-8FC6-F6F093DE22C5 (simzone_c) VM RESERVATIONS|0                             

